nautobot/utilities/forms/fields.py

```diff
@@ -1,26 +1,26 @@
 import csv
+from io import StringIO
 import json
 import re
-from io import StringIO
 
-import django_filters
 from django import forms
 from django.apps import apps
 from django.conf import settings
 from django.contrib.contenttypes.models import ContentType
 from django.contrib.postgres.forms import SimpleArrayField
 from django.core.exceptions import MultipleObjectsReturned, ObjectDoesNotExist, ValidationError
 from django.db.models import Count, Q
-from django.forms.fields import BoundField, JSONField as _JSONField, InvalidJSONInput
+from django.forms.fields import BoundField, InvalidJSONInput
+from django.forms.fields import JSONField as _JSONField
 from django.urls import reverse
+import django_filters
+
+from nautobot.extras import utils as extras_utils
+from nautobot.utilities import choices as utilities_choices
+from nautobot.utilities import utils, validators
+from nautobot.utilities.forms import constants, widgets
+from nautobot.utilities.forms import utils as forms_utils
```

Comment:
Not `utilities_forms_utils`? :-)

Reply:
I've been trying to roll these up into `utilities.forms`. Still needs another pass to catch the rest of the stuff the script missed

Reply:
Also trying to avoid aliasing within an app. So the `nautobot.utilities` aliases will be going away